### ISSUE TITLE
example is not working

### ISSUE THREAD

**Issue by xxholly32** [Jinjiang/vue-mark-display#1]
**Describe the bug**
it`s not working on `npm run example/example:build`

browser error
```
Error: Woops, there was an error making the request.
    at XMLHttpRequest.xhr.onerror (util.js:154)
```

**To Reproduce**
1. git clone you demo
2. npm install / yarn install
3. `run example` error tell me to add favicon so i did
4. `run example` enter localhost:1234, just have one page and error message, 

**Expected behavior**
working with your webcome.md file

**Screenshots**
![image](https://user-images.githubusercontent.com/6063358/59192284-83049300-8bb4-11e9-93a6-ce4448446ff1.png)

**Desktop (please complete the following information):**
 - OS: windows
 - Browser chrome
 - Version master

**Additional context**
i change your dependencies to remove all "^", 

npm install and example

working~~~~ i dont know why .

**Comment by Jinjiang** [comment]
Sorry I tried several times. Couldn't reproduce it. Could you please provide more information by:

1. Add `console.log(welcome)` below `examples/App.vue` line 20.
2. Run `yarn examples` or `npm run examples` again.
3. Open `http://localhost:1234/` and see what print out in the console.

On my Chrome for macOS, it print `/welcome.e911e96b.md`. Please have a check whether the result be like this.

Thanks.

**Comment by xxholly32** [comment]
i used chrome for macOS too

at first 

```
xiangxiaodeMacBook-Pro:vue-mark-display xiangxiao$ yarn examples
yarn run v1.17.3
$ parcel examples/index.html -d tmp
Server running at http://localhost:1234 
🚨  /Users/xiangxiao/Documents/work/workspace/vue-mark-display/examples/favicon.ico: ENOENT: no such file or directory, open '/Users/xiangxiao/Documents/work/workspace/vue-mark-display/examples/favicon.ico'
Error: ENOENT: no such file or directory, open '/Users/xiangxiao/Documents/work/workspace/vue-mark-display/examples/favicon.ico'
```

remove `ico` code

`examples/App/vue` line 20 was
```
<h1 id="welcome">Welcome</h1>
<p>this is a intro about <code>vue-mark-display</code></p>
<hr>
<h2 id="usage">Usage</h2>
....
```

i try many times , found markdown file can`t loader ,even in your [repo vue-mark-preview](https://github.com/Jinjiang/vue-mark-preview)

i fork and fix it with [commit](https://github.com/xxholly32/vue-mark-preview/commit/be6579cf92bb0040525442bcfdafeb5437f48615#diff-0e893b3578ae3d5c8b98bac1c80756a7R4)

just add file-loader ,or md file cant resolve

**Comment by Jinjiang** [comment]
Interesting. That might be a parceljs config issue. Will track this further. Thanks.

**Comment by Jinjiang** [comment]
fixed by the latest commit 17e57e7
